app/views/spree/admin/taxjar_settings/edit.html.erb

```diff
@@ -5,27 +5,70 @@
 <% end %>
 
 <% if ENV["TAXJAR_API_KEY"] %>
-  <h2>Nexus Regions</h2>
-  <table>
-  <thead>
-    <tr>
-      <th>Country Code</th>
-      <th>Country</th>
-      <th>Region Code</th>
-      <th>Region</th>
-    </tr>
-  </thead>
-  <% @nexus_regions.each do |region| %>
-    <tr>
-      <td><%= region.country_code %></td>
-      <td><%= region.country %></td>
-      <td><%= region.region_code %></td>
-      <td><%= region.region %></td>
-    </tr>
-  <% end %>
-  </table>
-  <a class="btn btn-primary" href="<%= spree.admin_taxjar_settings_sync_nexus_regions_path %>">Sync Nexus Regions</a>
-  <a class="btn btn-outline-primary" href="https://app.taxjar.com/account#states">Go to TaxJar to configure states</a>
+  <fieldset>
+    <legend>Nexus Regions</legend>
+    <table>
+    <thead>
+      <tr>
+        <th>Country Code</th>
+        <th>Country</th>
+        <th>Region Code</th>
+        <th>Region</th>
+      </tr>
+    </thead>
+    <% @nexus_regions.each do |region| %>
+      <tr>
+        <td><%= region.country_code %></td>
+        <td><%= region.country %></td>
+        <td><%= region.region_code %></td>
+        <td><%= region.region %></td>
+      </tr>
+    <% end %>
+    </table>
+    <a class="btn btn-primary" href="<%= spree.admin_taxjar_settings_sync_nexus_regions_path %>">Sync Nexus Regions</a>
+    <a class="btn btn-outline-primary" href="https://app.taxjar.com/account#states">Go to TaxJar to configure states</a>
+  </fieldset>
+  <fieldset>
```

Comment:
We didn't do this for the previous section (nexus regions), but do you mind pulling this into a partial so it's easier to move around in the future 🙏🏼